src/main/java/moze_intel/projecte/network/packets/SyncEmcPKT.java

```diff
@@ -17,11 +17,11 @@
 
 public class SyncEmcPKT implements IMessage
 {
-	private int[][] data;
+	private long[][] data;
```

Comment:
if you feel like it, now would be a good time to make this strongly typed (a class with `int id`, `int damage`, `long data`) or whatever, instead of an ugly 2d array. not a big deal though.